aten/src/ATen/native/mps/operations/UnaryOps.mm

```diff
@@ -263,5 +270,33 @@ void unary_op(const Tensor& self, const Tensor& output, std::string op_name, Una
                 });
 }
 
+
+static bool mpsSupportsCumsum() {
```

Comment:
This also probably belongs to a single place.

Reply:
Yes, I believe it should be property of MPSDevice and run only once. Do you mind if I'll do it as followup PR?